### PR TITLE
Changed link titles in Google-Cloud section.

### PR DESCRIPTION
**Pull request by Mr-Destructive**
Changed the link titles in `google-cloud` section of tutorials.

#62

### CODE REVIEW

File: tutorials/google-cloud/_posts/2019-04-28-translation.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ To add the library to your classpath, add this maven dependency to your `pom.xml
 
 You can call the Cloud Translation library from any Java code, including standalone (non-server) code. To see how the library works, let's start with that.
 
-You can view this example project [here](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/google-cloud/google-cloud-example-projects/translation-hello-world-standalone), or download it as a `.zip` file from DownGit [here](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/google-cloud/google-cloud-example-projects/translation-hello-world-standalone).
+You can view this [hello world](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/google-cloud/google-cloud-example-projects/translation-hello-world-standalone) example project [, or download it as a `.zip` file from [DownGit](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/examples/google-cloud/google-cloud-example-projects/translation-hello-world-standalone).
```
Comment (KevinWorkman) [inline tutorials/google-cloud/_posts/2019-04-28-translation.md]: I think you have an extra `[` here?

File: tutorials/google-cloud/_posts/2020-09-06-oauth-2.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ Let's look at each piece in more detail.
 
 # GoogleAuthorizationCodeFlow
 
-The [AuthorizationCodeFlow](https://googleapis.dev/java/google-oauth-client/1.31.0/index.html?com/google/api/client/auth/oauth2/AuthorizationCodeFlow.html) class is the core of Google's OAuth library. You can use it to connect to any OAuth 2.0 server: [here](https://blog.overops.com/tutorial-how-to-implement-java-oauth-2-0-to-sign-in-with-github-and-google/) is an example that uses it to sign into GitHub.
+The [AuthorizationCodeFlow](https://googleapis.dev/java/google-oauth-client/1.31.0/index.html?com/google/api/client/auth/oauth2/AuthorizationCodeFlow.html) class is the core of Google's OAuth library. You can use it to connect to any OAuth 2.0 server: [GitHub Sigin tutorial](https://blog.overops.com/tutorial-how-to-implement-java-oauth-2-0-to-sign-in-with-github-and-google/) is an example that uses it to sign into GitHub.
```
Comment (KevinWorkman) [inline tutorials/google-cloud/_posts/2020-09-06-oauth-2.md]: Typo: "Sigin" should probably be "Sign In"?

File: tutorials/google-cloud/_posts/2020-10-11-migrating-to-java-11.md [hunk 3]
```diff
@@ -205,7 +205,7 @@ There are many ways to include your own server, and if you've heard about Java f
 
 I personally recommend using [Jetty](https://www.eclipse.org/jetty/) as your server, because A: it's what the Java 8 runtime used behind the scenes and B: I find it more obvious than more complicated frameworks.
 
-The [official example repo](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine-java11) includes a few examples for different approaches you could take, including [this one](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine-java11/helloworld-servlet) that uses Jetty to deploy a servlets-based web app.
+The [official example repo](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine-java11) includes a few examples for different approaches you could take, including an [hello world](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine-java11/helloworld-servlet) example that uses Jetty to deploy a servlets-based web app.
```
Comment (KevinWorkman) [inline tutorials/google-cloud/_posts/2020-10-11-migrating-to-java-11.md]: "an hello world example" should be "a hello world example"

**Review by KevinWorkman — CHANGES_REQUESTED** [review]

File: tutorials/google-cloud/_posts/2020-09-06-oauth-2.md [hunk 4]
```diff
@@ -146,7 +146,7 @@ Let's look at each piece in more detail.
 
 # GoogleAuthorizationCodeFlow
 
-The [AuthorizationCodeFlow](https://googleapis.dev/java/google-oauth-client/1.31.0/index.html?com/google/api/client/auth/oauth2/AuthorizationCodeFlow.html) class is the core of Google's OAuth library. You can use it to connect to any OAuth 2.0 server: [GitHub Sigin tutorial](https://blog.overops.com/tutorial-how-to-implement-java-oauth-2-0-to-sign-in-with-github-and-google/) is an example that uses it to sign into GitHub.
+The [AuthorizationCodeFlow](https://googleapis.dev/java/google-oauth-client/1.31.0/index.html?com/google/api/client/auth/oauth2/AuthorizationCodeFlow.html) class is the core of Google's OAuth library. You can use it to connect to any OAuth 2.0 server: [GitHub Sig In tutorial](https://blog.overops.com/tutorial-how-to-implement-java-oauth-2-0-to-sign-in-with-github-and-google/) is an example that uses it to sign into GitHub.
```
Comment (KevinWorkman) [inline tutorials/google-cloud/_posts/2020-09-06-oauth-2.md]: You're still missing an "n" here 🙂

"Sig In" should be "Sign In"

**Review by KevinWorkman — CHANGES_REQUESTED** [review]

File: tutorials/google-cloud/_posts/2020-09-06-oauth-2.md (hunk 4 above)
Comment (Mr-Destructive) [reply tutorials/google-cloud/_posts/2020-09-06-oauth-2.md]: I don't why I keep making silly typos :man_facepalming:

**Review by KevinWorkman — APPROVED** [review]
Thanks!

**Pull request merged**